kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/k8s/HelmClient.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.k8s;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import io.kroxylicious.systemtests.executor.Exec;
+import io.kroxylicious.systemtests.k8s.exception.KubeClusterException;
+
+import static java.util.Arrays.asList;
+
+/**
+ * The type Helm client.
+ */
+public class HelmClient {
+    private static final Logger LOGGER = LogManager.getLogger(HelmClient.class);
+
+    private static final String HELM_CMD = "helm";
+    private static final String HELM_3_CMD = "helm3";
+    private static final String INSTALL_TIMEOUT_SECONDS = "120s";
+    private static String helmCommand;
+    private String namespace;
+    private String version;
+
+    public HelmClient() {
+        if (!clientAvailable()) {
+            throw new KubeClusterException.NotFound("No helm client found on $PATH. $PATH=" + System.getenv("PATH"));
+        }
+    }
+
+    /**
+     * Client available boolean.
+     *
+     * @return the boolean
+     */
+    public static boolean clientAvailable() {
+        if (Exec.isExecutableOnPath(HELM_CMD)) {
+            helmCommand = HELM_CMD;
+            return true;
+        }
+        else if (Exec.isExecutableOnPath(HELM_3_CMD)) {
+            helmCommand = HELM_3_CMD;
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * Set the namespace of the helm client.
+     *
+     * @param namespace the namespace
+     * @return the helm client
+     */
+    public HelmClient namespace(String namespace) {
+        this.namespace = namespace;
+        return this;
+    }
+
+    /** Install a chart given its name, release name, version and values to override
+     * @param chartName the chart name
+     * @param releaseName the release name
+     * @param version the version
+     * @param valuesMap the values to be overridden in the chart, set as key,value
+     * @return the helm client
+     */
+    public HelmClient install(String chartName, String releaseName, String version, Map<String, String> valuesMap) {
+        LOGGER.info("Installing helm-chart {}", releaseName);
+        String values = valuesMap.entrySet().stream()
+                .map(entry -> String.format("%s=%s", entry.getKey(), entry.getValue()))
+                .collect(Collectors.joining(","));
+        this.version = version;
```

Comment:
you treat the namespace field and the version field in a different way, why?

Reply:
Because version is only applicable to a concrete chart that will be defined by the duple `chartName` and `releaseName` and namespace is related to the kubernetes namespace where to install the chart.